src/edu/ucsd/library/xdre/tab/ExcelSource.java

```diff
@@ -248,23 +254,49 @@ else if ( cache != null && cache.size() > 0)
 	                    List<String> validValue = CONTROL_VALUES.get(header);
 	                    if (validValue != null && validValue.size() > 0) 
 	                    {
-	                    	// validate cell with multiple values
-	                    	String[] vals2Valid = value.split("\\" + DELIMITER);
-	                    	for (String val : vals2Valid) 
-	                    	{
-	                    		String normVal = val.trim();
-								if (header.equalsIgnoreCase("Language"))
-									normVal = normalizeFieldValue(normVal, DELIMITER_LANG_ELEMENT);
-								else if (header.equalsIgnoreCase(SubjectTabularRecord.SUBJECT_TYPE))
-									// Subject Import: case insensitive subject header value
-									normVal = val.toLowerCase();
-
-								if (!validValue.contains(normVal)) {
-			    	                String existing = invalids.get(originalHeader);
-			    	                if ( existing == null )
-			    	                	invalids.put(originalHeader, val);
-			    	                else
-			    	                	invalids.put(originalHeader, existing + " " + DELIMITER + " " + val);
+	                        if (originalHeader.equalsIgnoreCase(BEGIN_DATE) || originalHeader.equalsIgnoreCase(END_DATE))
+	                        {
+	                            // validate date-time format
+	                            boolean validDate = false;
+	                            StringBuilder messageBuilder = new StringBuilder();
+	                            for (String dateFormatter : validValue)
+	                            {
+		                            try {
+		                                validateDateTime(dateFormatter, value);
+		                                validDate = true;
+		                                break;
+		                            } catch (IllegalArgumentException ex) {
+		                                String errorMessage = (messageBuilder.length() > 0 ? " | " : "")
+		                                    + buildErrorReportMessage(values.get(TabularRecord.OBJECT_ID), header, value, ex.getMessage());
+		                                messageBuilder.append(errorMessage);
+		                            }
+	                            }
+
+	                            if (!validDate)
+	                            {
+	                                invalids.put(originalHeader, messageBuilder.toString());
+	                            }
+	                        }
+	                        else
+	                        {
+		                    	// validate cell with multiple values
```

Comment:
indention/spacing of this section seems inconsistent too. It seems like there is 2, 4, and 8 space indention. Particularly the if/else if checking Language below.